lib/index.js

```diff
@@ -5,48 +5,48 @@ const open = require('opener');
 const yargs = require('yargs');
 const serve = require('./serve');
 
-yargs // eslint-disable-line no-unused-expressions
-  .usage('$0 <cmd> [args]')
+const argv = yargs // eslint-disable-line no-unused-expressions
```

Comment:
Aha, I can now since I reuse the constant now, I'll just hotfix that on master before pushing the new release.